src/Native/Unix/System.IO.Ports.Native/pal_termios.c

```diff
@@ -0,0 +1,482 @@
+    case 2500000:
+        return B2500000;
+#endif
+#ifdef B2000000
+    case 2000000:
+        return B2000000;
+#endif
+#ifdef B1500000
+    case 1500000:
+        return B1500000;
+#endif
+#ifdef B1152000
+    case 1152000:
+        return B1152000;
+#endif
+#ifdef B1000000
+    case 1000000:
+        return B1000000;
+#endif
+#ifdef B921600
+    case 921600:
+        return B921600;
+#endif
+#ifdef B576000
+    case 576000:
+        return B576000;
+#endif
+#ifdef B500000
+    case 500000:
+        return B500000;
+#endif
+#ifdef B460800
+    case 460800:
+        return B460800;
+#endif
+    case 230400:
+        return B230400;
+    case 115200:
+        return B115200;
+    case 57600:
+        return B57600;
+    case 38400:
+        return B38400;
+    case 19200:
+        return B19200;
+    case 9600:
+        return B9600;
+    case 4800:
+        return B4800;
+    case 2400:
+        return B2400;
+    case 1800:
+        return B1800;
+    case 1200:
+        return B1200;
+    case 600:
+        return B600;
+    case 300:
+        return B300;
+    case 200:
+        return B200;
+    case 150:
+        return B150;
+    case 134:
+        return B134;
+    case 110:
+        return B110;
+    case 75:
+        return B75;
+    case 50:
+        return B50;
+    }
+    return B0;
+}
+
+static int
+TermiosRate2Speed(speed_t brate)
+{
+    switch (brate)
+    {
+#ifdef B4000000
+    case B4000000:
+        return 4000000;
+#endif
+#ifdef B3500000
+    case B3500000:
+        return 3500000;
+#endif
+#ifdef B3000000
+    case B3000000:
+        return 3000000;
+#endif
+#ifdef B2500000
+    case B2500000:
+        return 2500000;
+#endif
+#ifdef B2000000
+    case B2000000:
+        return 2000000;
+#endif
+#ifdef B1500000
+    case B1500000:
+        return 1500000;
+#endif
+#ifdef B1152000
+    case B1152000:
+        return 1152000;
+#endif
+#ifdef B1000000
+    case B1000000:
+        return 1000000;
+#endif
+#ifdef B921600
+    case B921600:
+        return 921600;
+#endif
+#ifdef B576000
+    case B576000:
+        return 576000;
+#endif
+#ifdef B500000
+    case B500000:
+        return 500000;
+#endif
+#ifdef B460800
+    case B460800:
+        return 460800;
+#endif
+    case B230400:
+        return 230400;
+    case B115200:
+        return 115200;
+    case B57600:
+        return 57600;
+    case B38400:
+        return 38400;
+    case B19200:
+        return 19200;
+    case B9600:
+        return 9600;
+    case B4800:
+        return 4800;
+    case B2400:
+        return 2400;
+    case B1800:
+        return 1800;
+    case B1200:
+        return 1200;
+    case B600:
+        return 600;
+    case B300:
+        return 300;
+    case B200:
+        return 200;
+    case B150:
+        return 150;
+    case B134:
+        return 134;
+    case B110:
+        return 110;
+    case B75:
+        return 75;
+    case B50:
+        return 50;
+    }
+
+    return brate;
+}
+
+int32_t
+Termios_GetSpeed(int32_t fd)
```

Comment:
Same question for all of these... why have a newline between the return type and the name?